.ci/pipeline/ci.yml

```diff
@@ -103,6 +103,7 @@ jobs:
   - script: |
       . /usr/share/miniconda/etc/profile.d/conda.sh
       conda activate CB
+      conda install -q -y -c intel dpctl
```

Comment:
Add it to `conda create` command:
`conda create -q -y -n CB -c conda-forge -c intel python=$(PYTHON_VERSION) dal-devel mpich dpctl`

Reply:
Fixed